README.md

```diff
@@ -85,10 +85,10 @@ Here you can automatically send heartbeat messages, send a logon/logout, request
 
 ```yaml
 apiVersion: th2.exactpro.com/v1
-kind: Th2GenericBox
+kind: Th2Box
 spec:
   image-name: your.image.repo:42/your_image_name
-  image-version: 1.0.0
+  image-version: 2.0.3
```

Comment:
```suggestion
  image-version: 1.0.0
```
Revert this line, because this is the base docker image without protocol logic. It will be used in other repositories.